src/main/kotlin/no/skatteetaten/aurora/gobo/graphql/auroraconfig/AuroraConfig.kt

```diff
@@ -60,6 +62,14 @@ data class ApplicationDeploymentSpec(
     val replicas = rawJsonValueWithDefaults.int("/replicas/value")
     val paused = rawJsonValueWithDefaults.boolean("/pause/value")
     val affiliation = rawJsonValueWithDefaults.text("/affiliation/value")
+    val permissions = rawJsonValueWithDefaults.let {
+        kotlin.runCatching {
+            JsonPath.read<Map<String, Map<String, String>>>(it.toString(), "\$.permissions").map { permission ->
+                val values = permission.value["value"]?.split(" ")
```

Comment:
Denne måten å hente listeverdier er litt sårbart for endring, men ikke noe vi enkelt kan gjøre noe med.

Reply:
ja, den er veldig sårbar dessverre